lib/utils/template_generator.js

```diff
@@ -50,6 +50,9 @@ class TemplateGenerator {
   installTemplate(templatePath, name, installPackages) {
     utils.cd(templatePath);
     utils.sed('package.json', '%APP_NAME%', name);
+    if (fs.existsSync('dot.gitignore')) {
+      fs.moveSync('dot.gitignore', '.gitignore');
```

Comment:
Won't move make the file "disappear" form the global install? Making it impossible to install the template twice? Shouldn't you copy instead?

Reply:
It's only being moved inside the template instance. `templatePath` is the same as `fspath` a few lines above, which is derived from [destinationFolder](https://github.com/embark-framework/embark/blob/0d881f9867e820664df51403afa90880d92841c9/lib/utils/template_generator.js#L36).

I was, though, concerned about this very thing. So I manually checked that it's only getting moved inside the template instance, and that in embark's template dir the file is still `dot.gitignore`.

Reply:
Oh duh, you're right.